src/WebJobs.Script.WebHost/WebHostServiceCollectionExtensions.cs

```diff
@@ -65,9 +69,14 @@ public static void AddWebJobsScriptHost(this IServiceCollection services, IConfi
             // Standby services
             services.AddStandbyServices();
 
+            // Register common services with the WebHost
+            // Language Worker Hosted Services need to be intialized before WebJobsScriptHostService
+            ScriptHostBuilderExtensions.AddCommonServices(services);
```

Comment:
Is this move required to ensure hosted services added here are registered before the `WebJobsScriptHostService`? 

We might be better served by moving the `WebJobsScriptHostService` registration down and adding a comment there.